deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/worker/HotfixStateTransitioner.java

```diff
@@ -76,6 +76,8 @@ void processBatch() throws Exception {
             } catch (Throwable t) {
                 // Catch all throwable so that subsequent job not suppressed, also long error in DB
                 LOG.error("HotfixStateTransitioner failed to process {} " + hotfixId, t);
+                hotBean.setError_message("Get Exception: " + t);
```

Comment:
Do we show the error to the UI?

Reply:
Yea, I think this is the new requirement after that incident.